tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -103,10 +105,25 @@ func (c *cluster) bootstrap() error {
 		}
 	}
 
-	c.Stressers = make([]Stresser, len(members))
+	c.Stressers = make([]Stresser, 0)
+	c.LeaseStressers = make([]LeaseStresser, len(members))
+	// init stressers
 	for i, m := range members {
-		c.Stressers[i] = c.stressBuilder(m)
-		go c.Stressers[i].Stress()
+
+		lsStresser := c.leaseStresserBuilder(m, i)
```

Comment:
I saw the builder pattern is used for stresser. I merely model the setup of lease stresser from stresser. I wonder should I be consistent?

https://github.com/fanminshi/etcd/blob/master/tools/functional-tester/etcd-tester/main.go#L76-L80
https://github.com/fanminshi/etcd/blob/master/tools/functional-tester/etcd-tester/cluster.go#L106-L110